pandas/tests/indexes/test_multi.py

```diff
@@ -1717,6 +1725,12 @@ def test_from_tuples(self):
                                'from empty list',
                                MultiIndex.from_tuples, [])
 
+        result = MultiIndex.from_tuples([], names=['a', 'b'])
```

Comment:
No, IMO this should not work here, as in this case you actually do pass 2 values, but empty values. So IMO this should raise.